lmdk/readme.txt

```diff
@@ -0,0 +1,13 @@
+To build dummy loadable library:
+
```

Comment:
Needs to be called README.md for GH to render.

Reply:
Fixed.